src/dso_api/dynamic_api/filterset.py

```diff
@@ -45,3 +63,8 @@ def filterset_factory(model: Type[DynamicModel]) -> Type[DynamicFilterSet]:
     }
     meta = type("Meta", (), meta_attrs)
     return type(f"{model.__name__}FilterSet", (DynamicFilterSet,), {"Meta": meta})
+
+
+def _get_lookups(field_cls) -> list:
```

Comment:
Het is zal mooier zijn om deze `_get_field_lookups` te noemen en als een `@classmethod` op `DynamicFilterSet` te zetten. e.g.

```
@classmethod
def _get_field_lookups(cls, field) -> list:
    """.."""
    return DEFAULT_LOOKUPS_BY_TYPE.get(field.__class__, ["exact"])
```

Dan later we kan deze method makkelijk overschrijven.

Reply:
> Dan later we kan deze method makkelijk overschrijven.

Dt kan helaas neit, die methode wordt gebruikt in de factory om een `DynamicFilterSet` subclass te maken. Bij het opbouwen van de `DynamicFilterSet.Meta` class worden deze 'fields' al bepaald.

Reply:
agh, ik had gedacht dat dit is `DSOFilterSet` class. Excuus.

Maar....  `field` ipv `field_cls` accepteren is nog steeds beter: 
  Later kan ik field._meta gebruiken on extensieve lookups voor de arrays aanmakken.

Reply:
ah, tx! dat is een goede suggestie. Dat heb ik nog wel aangepast.